### ISSUE TITLE
'custombadchars' doesn't work

### ISSUE THREAD

**Issue by MagmaChipmunk** [Nandaka/PixivUtil2#925]
### Prerequisites

* [x] Did you read FAQ section in [readme.md](https://github.com/Nandaka/PixivUtil2/blob/master/readme.md)?
* [x] Did you test with the latest [releases](https://github.com/Nandaka/PixivUtil2/releases "latest releases") or [commit](https://github.com/Nandaka/PixivUtil2/commits/master "latest commit") ?
* [x] Did you search for existing issues in [Issues](https://github.com/Nandaka/PixivUtil2/issues)?

### Description

I entered a value in 'custombadchars', but it doesn't work.

### Steps to Reproduce

1. input value 'custombadchars = %pattern<1>(maze)%%replace<1>(labyrinth)%'
2. The file name is changed and saved as follows
dir : artists (123456) -> artists \_\_23456\_
filename :
123456_p0_N47yJtW06PKwZP6zvdY7CqG1_maze.jpg
-> \_23456__0_N47_J_W06PKwZP6_vdY7CqG______.j_g

**Expected behavior:** [What you expected to happen]
maze -(replace to)> labyrinth

**Actual behavior:** [What actually happened]
All characters entered in 'custombadchars' are converted to '_'

### Versions

20210221-beta2 (latest releases)

You can get this information from executing `PixivUtil2.py --help`.
Latest version available in https://github.com/Nandaka/PixivUtil2/releases

**Comment by bluerthanever** [comment]
hmm. sorry for the inconvenience. let me have a look.

**Comment by bluerthanever** [comment]
hmm. it looks like the enhanced `customBadChars` is not yet available in `20210221-beta2`.
the pull request was merged on 9th Mar.

btw i just tried running from scripts, and it seems to be working fine...
my settings are:
```
...
[Filename]
filenameformat = %member_id%\%image_id% - %title%
filenamemangaformat = %member_id%\%image_id% - %title%\%urlFilename%
filenameinfoformat = %member_id%\%image_id% - %title%
filenamemangainfoformat = %member_id%\%image_id% - %title%\%image_id% - %title%
filenameseriesjson = %artist% (%member_id%)\%manga_series_id% - %manga_series_title%
filenameformatsketch = %artist% (%member_id%)\%urlFilename% - %title%
filenameformatnovel = %artist% (%member_id%)\%manga_series_id% %manga_series_order% %urlFilename% - %title%
avatarnameformat = 
backgroundnameformat = 
tagsseparator = ,
createmangadir = False
usetagsasdir = False
urldumpfilename = url_list_%Y%m%d
usetranslatedtag = False
tagtranslationlocale = en
custombadchars = %pattern<1>(オリ)%%replace<1>(labyrinth)%
...
```
and i tried to download image `123456`, the name was changed to `123456 - フlabyrinthイ`

please wait for the next release?

**Comment by bluerthanever** [comment]
you are welcome. glad to help. XD

i‘m just too lazy to make test files, and i usually test with actual data or downloading an existing image or something.... don't worry i got you.
